cellprofiler/gui/moduleview.py

```diff
@@ -694,7 +694,8 @@ def on_select_shape(event, setting=v, control=shape_control):
             )
 
             def on_set_size(event):
-                v.size = int(event.GetString())
+                size = event.GetString()
+                v.size = int(size) if size else ""
```

Comment:
Empty string for the alternate? I would assume it would be something like `0` or `None`.